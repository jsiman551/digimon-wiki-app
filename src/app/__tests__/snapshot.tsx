import Home from "../page";
import { render } from "@testing-library/react";

it("renders Home component", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
