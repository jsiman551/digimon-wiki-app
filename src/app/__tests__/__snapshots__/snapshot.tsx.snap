// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<div>
  <main
    data-testid="homeContainer"
  >
    <button
      class="btn btn-primary"
      type="button"
    >
      Primary
    </button>
     
    <button
      class="btn btn-secondary"
      type="button"
    >
      Secondary
    </button>
     
    <button
      class="btn btn-success"
      type="button"
    >
      Success
    </button>
     
    <button
      class="btn btn-warning"
      type="button"
    >
      Warning
    </button>
     
    <button
      class="btn btn-danger"
      type="button"
    >
      Danger
    </button>
     
    <button
      class="btn btn-info"
      type="button"
    >
      Info
    </button>
     
    <button
      class="btn btn-light"
      type="button"
    >
      Light
    </button>
     
    <button
      class="btn btn-dark"
      type="button"
    >
      Dark
    </button>
    <button
      class="btn btn-link"
      type="button"
    >
      Link
    </button>
  </main>
</div>
`;
